lib/Runtime/Library/GlobalObject.cpp

```diff
@@ -617,7 +617,15 @@ namespace Js
                 Throw::FatalInternalError();
             }
 #endif
+
+#if ENABLE_TTD
+            if(!scriptContext->IsTTDRecordOrReplayModeEnabled())
+            {
+                scriptContext->AddToEvalMap(key, isIndirect, pfuncScript);
```

Comment:
We disabled the cache during record (since trying to replay the behavior was problematic) so no need to reset cache values here.